README.md

```diff
@@ -1,3 +1,55 @@
 # jEntityTest
 [![CircleCI](https://circleci.com/gh/TestMonkeys/jEntityTest/tree/develop.svg?style=svg)](https://circleci.com/gh/TestMonkeys/jEntityTest)
 [![codecov](https://codecov.io/gh/TestMonkeys/jEntityTest/branch/master/graph/badge.svg)](https://codecov.io/gh/TestMonkeys/jEntityTest)
+
+jEntityTest is a library that can compare two instances of a class based on their declared fields. Making it easier to check if they are equal. It also contains a hamcrest matcher so that you can use the comparison in assertions. The comparison can further be customized via annotations for each field.
+
+# Usage Example
+
+A simple use case would be creating a user via API, then retrieving it and checking it was created correctly:
+```java
+User createdUser = api.createUser(newUser);
+assertThat(createdUser, Entity.isEqualTo(newUser));
+```
+In the above case, `Entity.isEqualTo`, which is part of jEntityTest, will do the comparison between the two objects. 
+Now let's look at the actual fields that the User object has
+
+User fields  | newUser values | createdUser values 
+------------ | -------------- | ------------------ 
+id           |                | 1
+firstName    | John           | John
+lastName     | Doe            | Doe
+
+The id was generated on while creating the user, so we can't know it beforehand. For the comparison to not fail in this case - fields can be decorated with annotations specifying the comparison strategy to be applied, so the user model would be:
+```java
+
+public class User{
+
+  @IgnoreComparison
+  private string id;
+  
+  private string firstName;
+  
+  private string lastName;
```

Comment:
did you mean `String`? 
also what about to provide parametrized comparison for one on String fields? like
`@StringComparison(caseSensitive = false)`
just for example of parametrized comparison

Reply:
yep, Should be String, more about parameterized comparison will be added in wiki